src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -674,6 +674,16 @@ public virtual bool SupportsBreakpointChecksums()
             return false;
         }
 
+        public virtual Task<bool> RequiresOnKeywordForBreakInsert()
+        {
+            return Task<bool>.FromResult(false);
+        }
+
+        public async Task<string> GetVersion()
+        {
+            return await _debugger.ConsoleCmdAsync("version");
```

Comment:
you should make this abstract and then no-op in non `lldb` cases. this will fail in `gdb` so if you want  to do it, then also implement the correct command in `gdb`.

```
(gdb)
interpreter-exec console version
&"interpreter-exec console version\n"
Undefined command: "version".  Try "help".
&"error in command: \"version\".\n"
^error,msg="error in command: \"version\"."
(gdb)
version
&"version\n"
&"Undefined command: \"version\".  Try \"help\".\n"
^error,msg="Undefined command: \"version\".  Try \"help\"."
(gdb)
```